ciphers/decrypt_caesar_with_chi_squared.py

```diff
@@ -0,0 +1,225 @@
+def decrypt_caesar_with_chi_squared(
+    ciphertext: str,
+    cipher_alphabet: list = [],
+    frequencies_dict: dict = {},
```

Comment:
```suggestion
    cipher_alphabet = None,
    frequencies_dict = None,
```
Mutable Default Arguments https://docs.python-guide.org/writing/gotchas/